client/src/components/MultiJurisdictionAudit/Setup/Review/index.tsx

```diff
@@ -11,26 +11,25 @@ import useAuditSettings from '../../useAuditSettings'
 import useContests from '../../useContests'
 import { IContest, ISampleSizeOption } from '../../../../types'
 import useJurisdictions, { FileProcessingStatus } from '../../useJurisdictions'
-import { api, checkAndToast, testNumber } from '../../../utilities'
+import { testNumber } from '../../../utilities'
```

Comment:
<img width="546" alt="Screen Shot 2020-07-20 at 1 07 23 PM" src="https://user-images.githubusercontent.com/530106/87981367-ffd99000-ca89-11ea-8599-4d8fdda935a5.png">

was playing around and triggered this error message (I think by typing a number and then deleting it). i think it's a pretty confusing message to a user

Reply:
@jonahkagan Hmmm... we should look at how we're handling that in the other validations. I think it crops up more than here, and will require a solution across the board.